pravega/pravega
# Issue 5793: Add root cause to bug report template.

Signed-off-by: Flavio Junqueira (fpj) <fpj@apache.org>

**Change log description**  
* Adds a comment about describing the root cause in the bug report issue template.

**Purpose of the change**  
Fixes #5793 

**What the code does**  
There is no code change, only a change to issue template.

**How to verify it**  
The change does not affect the build, it should be reflected in the bug report template.
